allzpark/_rezapi.py

```diff
@@ -30,12 +30,45 @@ def clear_caches():
         repo.clear_caches()
 
 
-def find_one(name, range_=None, paths=None):
-    return next(find(name, range_, paths))
+def find_one(name, range_=None, paths=None, package_filter=None):
+    '''
+    Find next package version
+
+    Args:
+        name (str): Name of the rez package
+        range_ (VersionRange or str, optional): Limits versions to range
+        paths (list of str, optional): Paths to search for packages
+        package_filter (PackageFilter, optional): Limits versions to those
+                                                  that match package filter
+
+    Returns:
+        rez.packages_.Package
+    '''
+    if package_filter:
+        return next(package_filter.iter_packages(name, range_, paths))
+    else:
+        return next(find(name, range_, paths))
+
+
+def find_latest(name, range_=None, paths=None, package_filter=None):
+    '''
+    Find latest package version
+
+    Args:
+        name (str): Name of the rez package
+        range_ (VersionRange or str, optional): Limits versions to range
+        paths (list of str, optional): Paths to search for packages
+        package_filter (PackageFilter, optional): Limits versions to those
+                                                  that match package filter
+
+    Returns:
+        rez.packages_.Package
+    '''
```

Comment:
This may sound picky, but this project mostly use double quote, so would be great if you can make that cosmetic chagne for those docstring. ☺️

Reply:
Seconded! :D But yes, nice find and great work @evil-g